### PR TITLE
Disable code signing

### PR DESCRIPTION
**Pull request by heyams**

### CODE REVIEW

File: .pipelines/pipeline.user.windows.yml [hunk 1]
```diff
@@ -12,9 +12,10 @@ versioning:
     - !!defaultcommand
       name: 'Generate Version'
       command: '.scripts/version.cmd'
+# excluding our source code from being scanned for code sign validation
 signing_options:
-  profile: '170'
-  codesign_validation_glob_pattern: 'file|**/agent/agent/build/libs/applicationinsights-agent-*.jar'
+  profile: 'none'
+  codesign_validation_glob_pattern: 'f:regex|\.(jar|pom)$'  # recursively find files ending in .jar, .pom
```
Comment (trask) [inline .pipelines/pipeline.user.windows.yml:18]: is the pattern still needed for profile `none`?

**Review by trask — APPROVED** [review]

File: .pipelines/pipeline.user.windows.yml (hunk 1 above)
Comment (heyams) [reply .pipelines/pipeline.user.windows.yml]: i have one of the project admins review it as well.. he offered some feedback.. it's updated.

Comment (heyams) [reply .pipelines/pipeline.user.windows.yml]: i will testing it without regex pattern.. once i'm done testing this change.

**Comment by heyams** [comment]
profile: 'none'
codesign_validation_glob_pattern: 'f:regex|\.(jar|pom)$'

@trask this skipped code signing.. not sure about the new change yet.. testing it now.

**Comment by heyams** [comment]
@trask Testing Oyster's feedback and removing pattern entirely.. both worked.  but without pattern is a bit faster.. saving ~7mins.

**Pull request merged**